app/controllers/games/pieces_controller.rb

```diff
@@ -6,7 +6,7 @@ def show
   def update
     @piece = Piece.find(params[:id])
     @piece.update_attributes(piece_params)
-    redirect_to game_path(@piece.game_id)
+    #redirect_to game_path(@piece.game_id)
```

Comment:
I think we are fine to remove this line completely. jQuery should handle the redirecting from now on.

Reply:
Removed!